### DISCUSSION TITLE
My college didn't give out personal mail

### DISCUSSION THREAD

**Discussion by Vladimir6752** [community/community#55260] [GitHub Education]
### Hi everyone,

Hello, I am a student from Russia, I sent an application to Github Education, but it was not accepted. They swear at a non-real name in the Github profile, my nickname is Vladimir6752, but my real name is Vladimir Babin, it's like that. As a document, I first sent a scan of my student ID, was not accepted, then sent a high-quality photo, but Github Education says that there is no name of my college there. They also told me, "Apparently, you are not near any location of the campus of your chosen school," why did they decide that? I study at college, not at home

**Comment by samus-aran** [comment]
Hi @Vladimir6752 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

In order to qualify for the GitHub education program, we must be able to verify that you are either located close to your campus, or that you are a part of a verifiable distance learning program.

You will have the most success applying if you are able to do so from your school’s campus.

- When asked, please enable the application to discover your location from the browser pop-up. Since settings for every browser are different, if you are having difficulty enabling us to see your location you should seek assistance from your browser's support pages.
- If you are still having difficulties enabling your location check out [this discussion on assessing your hardware or operating system's settings.](https://github.com/orgs/community/discussions/42246)
- If you are using a VPN, please disable it when you reapply. 
- If you are not using a VPN, try reapplying when connected to the internet from a different network.

Hope this helps, let us know how you get on 👍
